lineman/app/components/discussion_form/discussion_form.coffee

```diff
@@ -1,11 +1,11 @@
 angular.module('loomioApp').factory 'DiscussionForm', ->
   templateUrl: 'generated/components/discussion_form/discussion_form.html'
   controller: ($scope, $controller, $location, discussion, CurrentUser, Records, AbilityService, FormService, KeyEventService) ->
-    $scope.showGroupSelect = true
+    $scope.showGroupSelect = false
```

Comment:
Should be able to just drop this line; it'll automatically start as undefined which is falsey